File: algorithms/c/26.c
#include <stdio.h>

int removeDuplicates(int* nums, int numsSize)
{
    if ( numsSize == 0 )
        return 0;
    int index = 1;
    for ( int i = 1; i < numsSize; i++ )
        if ( nums[i] != nums[i-1] )
            nums[index++] = nums[i];
    return index;
}

int main()
{
    return 0;
}

File: algorithms/python3/26.py
class Solution:
    def removeDuplicates(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        l = 1
        for i in range(1, len(nums)):
            if nums[i] != nums[l-1]:
                nums[l] = nums[i]
                l += 1
        return l
